### ISSUE TITLE
Background apps

### ISSUE THREAD

**Issue by mburucuyapy** [linuxmint/webapp-manager#319]
Im sure if this is possible to implement. But give an option to run apps in the background. This would be great for certain messaging apps.

**Comment by viharm** [comment]
Yes, this would be very useful. An alternative would be to minimize to system tray.
